README.md

```diff
@@ -1 +1,2 @@
 # test-wca
```

Comment:
```diff
 # test-wca
+trigger
\ No newline at end of file
```

There are no issues with the changes. The `trigger` line has been added correctly. The `pass` command is not needed in this context.